README.md

```diff
@@ -819,17 +819,18 @@ In JSON land, that would give you the following structure.
 
 You can create nested arrays of tables as well. Just use the same double bracket
 syntax on sub-tables. Each double-bracketed sub-table will belong to the most
-recently defined table element above it.
+recently defined table element above it. Normal sub-tables (not arrays) likewise
```

Comment:
This is self-evident from the example JSON below, so I don't think we should call this out explicitly. None the less, if you think we should, let's tackle that in a separate PR?

Same for the `# subtable` and `# nested` comments below.